style/style.css

```diff
@@ -0,0 +1,743 @@
+    font-size: 1.5rem;
+  }
+
+  .menu_list li {
+    display: none;
+  }
+
+  .header .logo {
+    margin-left: 2.5em;
+    font-size: 1.5rem;
+  }
+
+  .desktop {
+    display: grid;
+    grid-template-columns: repeat(12, [col-start] 1fr);
+    column-gap: 24px;
+    background-image: url(../images/hero-background.svg);
+    background-size: contain;
+    height: 40rem;
+  }
+
+  .desktop .main {
+    display: grid;
+    grid-column: col-start 3 / span 8;
+  }
+
+  .main .main_item {
+    font-size: 3rem;
+    line-height: 5rem;
+    font-weight: 600;
+    margin-left: 7rem;
+    margin-top: 11rem;
+  }
+
+  .item1 {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+  }
+
+  .card_item .item1 {
+    width: 78rem;
+    height: 108rem;
+    display: flex;
+    flex-direction: column;
+    margin: auto;
+    margin-left: 50rem;
+  }
+
+  .card2 .item1_img {
+    padding-top: 18rem;
+  }
+
+  .card4 .item1_img {
+    padding-top: 18rem;
+  }
+
+  .card_item .item1 .item1_img {
+    width: 78rem;
+    margin-top: -9rem;
+  }
+
+  .item1 .item1_tool h2 {
+    font-size: 6rem;
+    margin: 4rem 0 0 -18rem;
+  }
+
+  .item1 .item1_tool h3 {
+    font-size: 3rem;
+    margin: 5rem 0 0 -17rem;
+  }
+
+  .item_list .item1_list {
+    font-size: 2rem;
+    margin: -1.7rem;
+    margin-left: -4rem;
+  }
+
+  .item_detail p {
+    font-size: 1.5rem;
+    line-height: 33px;
+    font-weight: 500;
+    width: 60rem;
+    margin-left: -17rem;
+  }
+
+  .languages {
+    margin-left: -15rem;
+  }
+
+  .item1_btn {
+    padding: 1rem 0;
+    text-align: center;
+    margin-left: -17rem;
+    margin-top: 5rem;
+    font-size: 21px;
+  }
+
+  .card2 .item_list .item2_list {
+    font-size: 2rem;
+    margin: -1rem;
+    margin-left: 0.5rem;
+  }
+
+  .card3 .item_list .item3_list {
+    font-size: 2rem;
+    margin: -1rem;
+    margin-left: 0.5rem;
+  }
+
+  .card4 .item_list .item4_list {
+    font-size: 2rem;
+    margin: -1rem;
+    margin-left: -8.5rem;
+  }
+
+  .about {
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-start;
+    width: 95rem;
+  }
+
+  .about .about_content h4 {
+    font-size: 6rem;
+  }
+
+  .about .about_content p {
+    font-size: 20px;
+    width: 63rem;
+    font-weight: 600;
+  }
+
+  .about .about_content span {
+    font-size: 2rem;
+  }
+
+  .about .about_content .icons a {
+    font-size: 2rem;
+  }
+
+  .about .about_content .about_btn {
+    font-size: 2rem;
+    padding: 1rem;
+    margin-top: 2rem;
+  }
+
+  .about .languages {
+    margin-left: 4rem;
+  }
+
+  .languages .lan_arrow h5 {
+    font-size: 3rem;
+  }
+
+  .about_work .fas {
+    font-size: 2rem;
+  }
+
+  .about_skill .fas {
+    font-size: 2rem;
+  }
+
+  .languages .lan_arrow .fas {
+    font-size: 2rem;
+  }
+
+  .about_work h6 {
+    font-size: 3rem;
+    margin-top: 8rem;
+  }
+
+  .about_skill h6 {
+    font-size: 3rem;
+  }
+
+  .contact {
+    width: 105rem;
+    margin-top: -4rem;
+  }
+}
```

Comment:
Great job 👍🏿  incorporating a responsive breakpoint for the desktop view in your project. However, I observed that the project's responsiveness is compromised when viewed on mobile devices compared to its previous state. To address this issue, I recommend implementing an additional breakpoint specifically designed for mobile view between `768px and below`. By doing so, you will ensure that your project meets the required responsiveness standards.